app/controllers/draws_controller.rb

```diff
@@ -99,10 +100,12 @@ def students_update
     handle_action(**result)
   end
 
+  def lottery_confirmation; end
+
   def start_lottery
     @lottery_starter = DrawLotteryStarter.new(draw: @draw)
     result = @lottery_starter.start
-    handle_action(action: 'show', **result)
+    handle_action(path: lottery_confirmation_draw_path(@draw), **result)
```

Comment:
Why wouldn't use `action: 'lottery_confirmation'`? You're already calculating the metrics for this action as well.